Formula/gforth.rb

```diff
@@ -13,14 +13,34 @@ class Gforth < Formula
   end
 
   depends_on "libtool" => :run
+  depends_on "emacs" => :build
   depends_on "libffi"
   depends_on "pcre"
 
   def install
     cp Dir["#{Formula["libtool"].opt_share}/libtool/*/config.{guess,sub}"], buildpath
     ENV.deparallelize
     system "./configure", "--prefix=#{prefix}"
-    system "make" # Separate build steps.
+    system "make", "EMACS=#{HOMEBREW_PREFIX}/bin/emacs"
     system "make", "install"
+
+    elisp.install "gforth.el"
+    elisp.install "gforth.elc"
+  end
+
+  def caveats
```

Comment:
don't need the caveat. if you're building from source and not pouring the bottle, you should expect build time dependencies you may not want to keep